patterns/wireguard-with-cilium/README.md

```diff
@@ -1,27 +1,56 @@
 # Transparent Encryption with Cilium and Wireguard
 
-This pattern demonstrates Cilium configured in CNI chaining mode with VPC CNI and with Wireguard transparent encryption enabled on an Amazon EKS cluster.
+This pattern demonstrates Cilium configured in CNI chaining mode with the VPC CNI and with Wireguard transparent encryption enabled on an Amazon EKS cluster.
 
-- [Cilium CNI Chaining Documentation](https://docs.cilium.io/en/v1.12/gettingstarted/cni-chaining-aws-cni/)
-- [Cilium Wireguard Encryption Documentation](https://docs.cilium.io/en/v1.12/gettingstarted/encryption-wireguard/)
+- [Cilium CNI Chaining Documentation](https://docs.cilium.io/en/stable/installation/cni-chaining-aws-cni/)
+- [Cilium Wireguard Encryption Documentation](https://docs.cilium.io/en/stable/security/network/encryption-wireguard/)
+
+## Areas of Interest
```

Comment:
Key focal points - just a thought.

Reply:
`Focal Points` sounds good - short and sweet, easy to understand the intent

Reply:
updated in https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/1761/commits/143ea9da75088d0247ed997fb17e3b30cf84c08a